docs/pages/_includes/services/install.md

```diff
@@ -27,12 +25,66 @@ All `dcos {{ include.packageName }}` CLI commands have a `--name` argument allow
+**Note:** Alternatively, you can [install from the DC/OS web interface](https://docs.mesosphere.com/latest/deploying-services/install/). If you install {{ include.techName }} from the DC/OS web interface, the `dcos {{ include.packageName }}` CLI commands are not automatically installed to your workstation. They may be manually installed using the DC/OS CLI:
 
 ```bash
 dcos package install {{ include.packageName }} --cli
 ```
 
+## Alternate install configurations
+
+The following are some examples of how to customize the installation of your {{ include.techName }} instance.
+
+In each case, you would create a new {{ instance.techName }} instance using the custom configuration as follows:
+
+```bash
+$ dcos package install {{ include.packageName }} --options=sample-{{ include.serviceName }}.json
+```
+
+**Recommendation:** Store your custom configuration in source control.
+
+### Installing multiple instances
+
+By default, the {{ include.techName }} service installed with a service name of `{{ include.serviceName }}`. You may specify a custom name using a custom configuration as follows:
+
+```json
+{
+  "service": {
+    "name": "{{ include.serviceName }}-custom"
+  }
+}
+```
+
+When the above JSON configuration is passed to the `package install {{ include.packageName }}` command via the `--options` argument, the new service will be named `{{ include.serviceName }}-custom`.
+
+Multiple instances of {{ install.techName }} may be installed into your DC/OS cluster by customizing the name of each instance. For example, you might have one instance of {{ install.techName }} named `{{ include.serviceName }}-staging` and another named `{{ include.serviceName }}-prod`, each with its own custom configuration. Note that this name cannot be changed after initial install, you must instead uninstall and reinstall the service (backing up any data as necessary).
+
+#### Addressing multiple instances using the CLI
+
+After you've installed the service under a custom name, it may be accessed from all `dcos {{ package.packageName }}` CLI commands using the `--name` argument. By default, the `--name` value defaults to the name of the package, or `{{ include.packageName }}`.
+
+For example, if you had an instance named `{{ include.serviceName }}-dev`, you would invoke the following to perform a `pod list` command against it:
+
+```bash
+$ dcos {{ include.packageName }} --name={{ include.serviceName }}-dev pod list
+```
+
+### Virtual networks
+
+{{ include.techName }} supports deployment on virtual networks on DC/OS (including the `dcos` overlay network), allowing each container (task) to have its own IP address and not use port resources on the agent machines. This can be specified by passing the following configuration during installation:
+
+```json
+{
+  "service": {
+    "virtual_network_enabled": true
```

Comment:
@mpereira can you make sure this gets updated with the labels parameter as part of your work?